### PR TITLE
Add preference for default input value (#19).

### PR DESCRIPTION
**Pull request by Osmose**
@mythmon r?

This CoffeeScript switch with stuff is cray

### CODE REVIEW

File: CHANGELOG.MD [hunk 1]
```diff
@@ -12,6 +12,7 @@
 * Fix bug where clearing the input and entering "~/" with Helm-style
   shortcuts enabled did not automatically fill in the user's home
   directory.
+* Added option for the default input value on toggle.
```
Comment (mythmon) [inline CHANGELOG.MD]: Yay for keeping a changelog!

Comment (Osmose) [inline CHANGELOG.MD]: I hate keeping them but Atom shows a link to them if you have one and the original project had one SO IM OBLIGATED GAH

File: lib/advanced-open-file-view.coffee [hunk 2]
```diff
@@ -323,14 +335,18 @@ class AdvancedFileView extends View
...
+    suggestedPath = ''
+    switch atom.config.get("advanced-open-file.defaultInputValue")
+      when DEFAULT_ACTIVE_FILE_DIR
+        activePath = atom.workspace.getActiveTextEditor()?.getPath()
+        if activePath
+          suggestedPath = path.dirname(activePath) + path.sep
+      when DEFAULT_PROJECT_ROOT
+        projectPaths = atom.project.getPaths()
+        if projectPaths.length > 0
+          suggestedPath = projectPaths[0] + path.sep
```
Comment (mythmon) [inline lib/advanced-open-file-view.coffee]: Do you need to include a case here for DEFAULT_EMPTY, or is the goal to rely on the implicit "do nothing" behavior?

Comment (Osmose) [inline lib/advanced-open-file-view.coffee]: You have identified the goal. This way it doesn't just leave the old value if the `if` statements within each case fails.

**Comment by mythmon** [comment]
This looks correct to me. Coffeescript switch stuff looks nice. r+

**Pull request merged**